.github/workflows/documentation.yml

```diff
@@ -0,0 +1,114 @@
+name: Documentation
+
+# Trigger the documentation build for any push, pull request and any branch only deploy on next
```

Comment:
Trigger the documentation -> what part ? the build ? the upload ? a bit more specificity would help (I'd guess it's check that the documentation builds in this case).

Reply:
Is the new comments ok to you ?